### PR TITLE
DOCS: Fix incorrect pass phrase options references [1.1.1]

### LINKED ISSUE #13883: openssl(1) section "PASS PHRASE ARGUMENTS"
A number of utilites refer to is - for example "For more information about the format of arg see the PASS PHRASE ARGUMENTS section in openssl(1).". There is no such section in 1.1+ manuals. Master is still not up to date.

### PR DESCRIPTION
**Pull request by levitte**
There were a number of older style references to the pass phrase
options section, now streamlined with the current openssl(1).

Fixes #13883

### CODE REVIEW

File: doc/man1/ca.pod [hunk 1]
```diff
@@ -163,7 +163,7 @@ self-signed certificate.
 =item B<-passin arg>
 
 The key password source. For more information about the format of B<arg>
-see the B<PASS PHRASE ARGUMENTS> section in L<openssl(1)>.
+see L<openssl(1)/Pass Phrase Options>.
```
Comment (kroeckx) [inline doc/man1/ca.pod:166]: According to the perl documentation, it requires quotes when using spaces.

File: doc/man1/ts.pod [hunk 2]
```diff
@@ -242,7 +242,7 @@ The name of the file containing a DER encoded timestamp request. (Optional)
 =item B<-passin> password_src
 
 Specifies the password source for the private key of the TSA. See
-B<PASS PHRASE ARGUMENTS> in L<openssl(1)>. (Optional)
+the B<PASS PHRASE ARGUMENTS> in L<openssl(1)>. (Optional)
```
Comment (t8m) [inline doc/man1/ts.pod:245]: The reference is still unfixed here

Comment (kroeckx) [inline doc/man1/ts.pod:245]: Forgot to change this?

File: doc/man1/ca.pod (hunk 1 above)
Comment (kroeckx) [reply doc/man1/ca.pod:166]: I got confused

File: doc/man1/ts.pod (hunk 2 above)
Comment (levitte) [reply doc/man1/ts.pod:245]: Kinda...  my sed script wasn't calibrated for this variation, and I missed that it had missed this one.  Fixed

Comment (levitte) [reply doc/man1/ts.pod:245]: Fixed

**Review by kroeckx — APPROVED** [review]

**Comment by levitte** [comment]
Merged

a83690c0b61e342f35a9583868b74e7ff6023101 DOCS: Fix incorrect pass phrase options references

**Pull request closed without merging**